azavea/echo-locator
# Update how the app lists neighborhood data

## Overview

This PR:
- removes the auth check from the neighborhood endpoints
- lists neighborhood data when auth component mounts
- adds `AWS_S3_PHOTO_BUCKET` env var to the local dotenv file sample

### Checklist


## Testing Instructions

- `./scripts/update` 
- `./scripts/server`
- Follow the steps in the linked issue that used to trigger the empty list and map state. Make sure the bug is fixed and make sure the neighborhood list result is as expected

Resolves https://github.com/azavea/echo-locator/issues/564
